src/utils/properties.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Gets an environment variable. If the env var is not set and a default value is not
+ * provided, then it is assumed it is a mandatory requirement and an error will be
+ * thrown.
+ */
+
+const getEnvironmentVariable = (key: string, defaultValue?: any): string => {
```

Comment:
defaultValue should probably be of type string, as its going to be returned as a string anyway.

Also, I think it may be better to have properties.ts directly under the src directory, removing the utils directory completely, will make it easier to find

Reply:
I spotted the default value yesterday and committed it... it just isn't showing on the PR? https://github.com/companieshouse/onboarding-portal-webapp/compare/refactor-env-vars

I can refactor the directories too, I agree it would be easier to see.